website/docs/d/integrations.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "runscope"
+page_title: "Runscope: runscope_integrations"
+sidebar_current: "docs-runscope-datasource-integrations"
+description: |-
+  Get information about runscope integrations enabled on for your team.
+---
+
+# runscope\_integration
+
+Use this data source to list all of your [integration]s(https://www.runscope.com/docs/api/integrations)
```

Comment:
Not sure if this `s` needs to be inside the brackets? or the URL next to the brackets?